include/sensor_fusion.h

```diff
@@ -4,9 +4,47 @@
  * \version 1.0
  * \date 2019-12-01
  */
+
 #ifndef SENSOR_FUSION_H
 #define SENSOR_FUSION_H
 
+/**
+ * \brief Structure for storing support degree matrix
+ * \details For the given number of sensors the structure will store support degree matrix
+ */
+struct support_degree_matrix {
+    int sensor_count;
+    double *sd_matrix;
+};
+
+/**
+ * \brief Structure for storing eigen value and eigen vector
+ * \details For a given support degree matrix of
+ * sensors the corresponding eigen values and vectors are stored in structure
+ */
+struct eigen_value_vector {
+    double *eigen_value;
+    double **eigen_vector;
+};
```

Comment:
See previous comment on using typedef

Reply:
fixed in the latest commit to follow typedef coding convention